site/public/js/forum.js

```diff
@@ -2531,12 +2495,6 @@ function pinAnnouncement(thread_id, type, csrf_token) {
             },
             // eslint-disable-next-line no-unused-vars
             success: function(data) {
-                if (type) {
-                    window.socketClient.send({'type': 'announce_thread', 'thread_id': thread_id});
-                }
-                else {
-                    window.socketClient.send({'type': 'unpin_thread', 'thread_id': thread_id});
-                }
             },
             error: function() {
```

Comment:
I know for other parts of the Submitty, when retrieving data goes wrong, we can see the error message in the discussion forum, can we have this for here in the discussion forum?